website/pages/docs/index.mdx

```diff
@@ -1,12 +1,33 @@
 import { Disclosures } from '../../components/home/faq';
-import { Tab, Tabs } from 'nextra-theme-docs';
+import { Callout, Tab, Tabs } from 'nextra-theme-docs';
+import Link from 'next/link';
+import dynamic from 'next/dynamic';
+
+export const Demo = dynamic(() =>
+  import('../../components/demo').then((mod) => mod.Demo),
+);
 
 # Introduction
 
 **Million.js** is an extremely fast and lightweight `<4kb` virtual DOM that makes React up to [70% faster](https://krausest.github.io/js-framework-benchmark/current.html). By using a fine-tuned, optimized virtual DOM, Million.js reduces the overhead of React.
 
 Imagine React components running at the speed of raw JavaScript.
```

Comment:
This sentence feels orphaned. Might be worthwhile finding a better home for it or adding something after

Reply:
Hmm, thanks for this Sarah! This is much appreciated. I really appreciate it 🤔. I just tried something on it.